src/Proxy.NET/Models/AttendeeApiKey.cs

```diff
@@ -0,0 +1,6 @@
+namespace Proxy.NET.Models;
+
+public partial class AttendeeApiKey
```

Comment:
Is there a reason this is a partial class?

Reply:
Punting to [#307](https://github.com/microsoft/azure-openai-service-proxy/issues/307) and #308